app/uk/gov/hmrc/plasticpackagingtax/registration/models/genericregistration/PartnerContactDetails.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.registration.models.genericregistration
+
+import play.api.libs.json.{Json, OFormat}
+import uk.gov.hmrc.plasticpackagingtax.registration.forms.contact.Address
+
+case class PartnerContactDetails(
```

Comment:
Same details are held in `GroupMemberContactDetails`. Is this not used to minimise disruption or some other reason?

Reply:
Not intentional. If we believed it was worth sharing then we'd obviously need to rename that class and relocate. You know a requirement for slightly different partner details will mysteriously surface if we go to the trouble of refactoring :grin:

Reply:
II'm suggesting we leave as is unless other people feel strongly that it should be changed.

Reply:
That's fine.